src/main/scripts/prepare_release.py

```diff
@@ -0,0 +1,183 @@
+import git
```

Comment:
Reading this script is not the easiest. I think it is ok as it is but maybe some readability finessing would be useful. For example, the short variable naming (without `_`) makes it harder to follow the code flow and it is not fully self-documenting. It would be also good to add a comment on the top describing the general functionality and processing flow.

Nevertheless, great job mate! This will save a lot of time and avoid errors.

Reply:
I agree about lack of docs and some readability. I'll make it a bit nicer.